addons/web_editor/static/src/js/backend/field_html.js

```diff
@@ -191,6 +198,7 @@ var FieldHtml = basic_fields.DebouncedField.extend(TranslatableFieldMixin, {
                 res_model: this.model,
                 res_id: this.res_id,
             },
+            enableResizer: this.withResizer,
```

Comment:
Why does this use two different variable names ? Why isn't it `this.enableResizer` ?

Reply:
Good point.